update slo audit event info (#23052)
DataDog/documentation: content/en/service_management/service_level_objectives/_index.md

> Three types of SLO audit events appear in the Event Explorer:

Reviewer: ```suggestion
#### Audit event categories

Three types of SLO audit events appear in the Event Explorer:
```

Reviewer: Suggesting this change for consistency with the format of the status correction audit events section

- `SLO Created` events show the SLO configuration information at creation time
- `SLO Modified` events show what configuration information changed during a modification
- `SLO Deleted` events show the configuration information the SLO had before it was deleted

> **SLO status correction audit events:**

Reviewer: ```suggestion
### Status correction audit events
```

Each event includes the following SLO status correction configuration information:

- SLO Name

> - Status correction start and end times and timezone

Reviewer: ```suggestion
- Status correction start and end times with timezone
```

> - Status correction category

Reviewer: ```suggestion
#### Status correction audit event categories
```

Author: This isn't a separate section, it's part of the list of the pieces of info in the event

Three types of SLO status correction audit events appear in the Event Explorer:

- `SLO Correction Created` events show the status correction configuration information at creation time
- `SLO Correction Modified` events show what configuration information changed during a modification
- `SLO Correction Deleted` events show the configuration information the status correction had before it was deleted

To get a full list of all SLO audit events, enter the search query `tags:(audit AND slo)` in the Event Explorer. To view the list of audit events for a specific SLO, enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO. You can also query the Event Explorer programmatically using the [Datadog Events API][19].

**Note:** If you don't see events appear in the UI, be sure to set the time frame of the Event Explorer to a longer period, for example, the past 7 days.

{{< img src="service_management/service_level_objectives/slo-audit-events.png" alt="SLO audit events" >}}

You can also use the "Audit History" tab in the SLO details to view all audit events for an individual SLO:

> {{< img src="service_management/service_level_objectives/slo_audit_history_tab.png" alt="SLO list page displays the Edit Tag dropdown for bulk tag editing" >}}

Reviewer: ```suggestion
{{< img src="service_management/service_level_objectives/slo_audit_history_tab.png" alt="SLO audit history tab" >}}
```

Reviewer: Feel free to make any additional edits to better describe this image

> With [Event Monitors][28], you can set up notifications to track SLO audit events. For example, if you wish to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`.

Reviewer: ```suggestion
With [Event Monitors][28], you can set up notifications to track SLO audit events. For example, if you want to be notified when a specific SLO's configuration is modified, set an Event Monitor to track the text `[SLO Modified]` over the tags `audit,slo_id:<SLO ID>`. Replace `<SLO ID>` with the SLO's ID.
```
